client/client.go

```diff
@@ -0,0 +1,32 @@
+package client
+
+import (
+	pstore "github.com/dell/gopowerstore"
+)
+
+// Client type is to hold powerstore client
+type Client struct {
+	PStoreClient *pstore.ClientIMPL
+}
+
+var (
+	newClientWithArgs = pstore.NewClientWithArgs
+)
+
+// NewClient returns the gopowermax client
```

Comment:
its a typo, it supposed to be gopowerstore client in a comment

Reply:
Updated